From 5ab83d0172b1a78b54ab473ebf9be639ffa8150d Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 5 Sep 2018 10:14:22 +0530
Subject: [PATCH 472/909] arm64: dts: r8a7795: Unsupport DMA transfer for SCIF2

commit 0632e46ff52160fee5161405be6747b1e9476703 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Since R8A7795 SoC is not supporting DMA transfer of SCIF2, this patch
deletes the property of "dmas" and "dma-names" from scif2 node.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index f86ad215bc9b..46e2ac535b2c 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -2,7 +2,7 @@
 /*
  * Device Tree Source for the R-Car H3 (R8A77950) SoC
  *
- * Copyright (C) 2015 Renesas Electronics Corp.
+ * Copyright (C) 2015-2017 Renesas Electronics Corp.
  */
 
 #include <dt-bindings/clock/r8a7795-cpg-mssr.h>
@@ -1560,9 +1560,6 @@
 				 <&cpg CPG_CORE R8A7795_CLK_S3D1>,
 				 <&scif_clk>;
 			clock-names = "fck", "brg_int", "scif_clk";
-			dmas = <&dmac1 0x13>, <&dmac1 0x12>,
-			       <&dmac2 0x13>, <&dmac2 0x12>;
-			dma-names = "tx", "rx", "tx", "rx";
 			power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
 			resets = <&cpg 310>;
 			status = "disabled";
-- 
2.17.1

